### PR TITLE
feat(lts): add new resource host group

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/lts' TESTARGS='-run TestAccHostGroup_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/lts -v -run TestAccHostGroup_basic -timeout 360m -parallel 4
=== RUN   TestAccHostGroup_basic
=== PAUSE TestAccHostGroup_basic
=== CONT  TestAccHostGroup_basic
--- PASS: TestAccHostGroup_basic (362.51s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/lts       362.581s
```

### CODE REVIEW

File: docs/resources/lts_host_group.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+subcategory: "Log Tank Service (LTS)"
+---
+
+# huaweicloud_lts_host_group
+
+Manages a LTS host group resource within HuaweiCloud.
```
Comment (Lance52259) [inline docs/resources/lts_host_group.md]:
```suggestion
Manages an LTS host group resource within HuaweiCloud.
```
The letter L is pronounced `[el]`
And check other places

File: huaweicloud/services/acceptance/lts/resource_huaweicloud_lts_host_group_test.go [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+  count = 2
+
+  name                = "%s-${count.index}"
+  description         = "terraform test"
+  image_id            = data.huaweicloud_images_image.test.id
+  flavor_id           = data.huaweicloud_compute_flavors.test.ids[0]
+  security_group_ids  = [data.huaweicloud_networking_secgroup.test.id]
+
+  network {
+    uuid              = data.huaweicloud_vpc_subnet.test.id
```
Comment (Lance52259) [inline huaweicloud/services/acceptance/lts/resource_huaweicloud_lts_host_group_test.go]:
```suggestion
    uuid = data.huaweicloud_vpc_subnet.test.id
```

Comment (Jason-Zhang9309) [reply huaweicloud/services/acceptance/lts/resource_huaweicloud_lts_host_group_test.go]: fixed

File: docs/resources/lts_host_group.md (hunk 1 above)
Comment (Jason-Zhang9309) [reply docs/resources/lts_host_group.md]: fixed

File: docs/resources/lts_host_group.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## Example Usage
+
+```hcl
+variable "group_name" {}
+variable "host_id_1" {}
+variable "host_id_2" {}
+
+resource "huaweicloud_lts_host_group" "test" {
+  name     = var.name
```
Comment (hmbig2) [inline docs/resources/lts_host_group.md]:
```suggestion
  name     = var.group_name
```

File: huaweicloud/services/lts/resource_huaweicloud_lts_host_group.go [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+	}
+
+	getHostGroupRespBody, err := utils.FlattenResponse(getHostGroupResp)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	jsonPath := fmt.Sprintf("result[?host_group_id=='%s']|[0]", d.Id())
+	getHostGroupRespBody = utils.PathSearch(jsonPath, getHostGroupRespBody, nil)
+	log.Printf("xxxx getHostGroupRespBody: %s", getHostGroupRespBody)
```
Comment (hmbig2) [inline huaweicloud/services/lts/resource_huaweicloud_lts_host_group.go]: remove this line

File: docs/resources/lts_host_group.md (hunk 3 above)
Comment (Jason-Zhang9309) [reply docs/resources/lts_host_group.md]: fixed

File: huaweicloud/services/lts/resource_huaweicloud_lts_host_group.go (hunk 4 above)
Comment (Jason-Zhang9309) [reply huaweicloud/services/lts/resource_huaweicloud_lts_host_group.go]: fixed

File: huaweicloud/services/acceptance/lts/resource_huaweicloud_lts_host_group_test.go [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+`, testHostGroup_base(name), name)
+}
+
+func testHostGroup_basic_update_2(name string) string {
+	return fmt.Sprintf(`
+%s
+
+resource "huaweicloud_lts_host_group" "test" {
+  name         = "%s-update"
+  type         = "linux"
```
Comment (Lance52259) [inline huaweicloud/services/acceptance/lts/resource_huaweicloud_lts_host_group_test.go]:
```suggestion
  name = "%s-update"
  type = "linux"
```

Comment (Jason-Zhang9309) [reply huaweicloud/services/acceptance/lts/resource_huaweicloud_lts_host_group_test.go]: fixed

**Pull request merged**